docker_mcp/server.py

```
@@ -350,22 +431,21 @@
 
     async def docker_compose(
```

Ruff (0.12.2)

`docker_compose` is too complex (34 > 10)

(C901)